packages/core/src/indexing/types.ts

```diff
@@ -4,7 +4,7 @@ import type { DID } from 'dids'
 export interface IndexStreamArgs {
   streamID: StreamID
   model: StreamID
-  controller: DID
+  controller: DID | string
```

Comment:
as Joel pointed out to me on a spec doc, the controller should probably always be a `string`, not a `DID` instance.  DID instances are only useful when you actually want access to the methods for signature creation/verification or encryption/decryption, if you just want a DID identifier it's best to always just use a string.

Reply:
I do kind of which there was a different type for a DID identifier though that enforced the string structure and provided accessors for the different parts of the DID string, but that doesn't exist atm as far as I know and isn't important enough to add now